portal/src/components/Communities/PopularList/PopularList.component.html

```diff
@@ -8,7 +8,7 @@
           class="popular-list__link"
           :to="localePath({name: 'communities-community', params: {community: community.id}})"
         >
-          <h3 class="popular-list__title">{{ getTitleTranslation(community, $i18n.locale) }}</h3>
+          <h3 class="popular-list__title">{{ getCommunityDetail(community, $i18n.locale, 'title') }}</h3>
```

Comment:
Wat is de reden dat `title` een argument is? Deze functie wordt maar 1 keer gebruikt toch?
En die `$i18n.locale` kan ook gewoon in de js-code door `this.$18n`. Dat zou ik eerder doen, zo min mogelijk in de HTML.

Reply:
je hebt helemaal gelijk. ik had de functie ergens anders vandaan gehaald.